ledger-tool/tests/basic.rs

```diff
@@ -0,0 +1,60 @@
+use solana::ledger::{create_tmp_genesis, create_tmp_sample_ledger, LedgerWriter};
+use solana_sdk::signature::{Keypair, KeypairUtil};
+use std::process::Command;
+use std::process::Output;
+use std::sync::Arc;
+
+static TARGET_PATH: &str = "../target/debug/solana-ledger-tool";
```

Comment:
It probably only worked because the debug build came first.  I've modified the test to use assert_cmd.  The first time this test executes it takes a long time and kicks off a bunch of compilations, so it's possible I'm not using assert_cmd correctly, but I'm guessing that's just how main_binary works.

Reply:
I don't see the use of `assert_cmd`. Did you push the commit?

Reply:
Oops.  I amended and pushed the commit but didn't add the latest changes.